### PR TITLE
Implemented sceUtilityGetSystemParamInt

### PR DESCRIPTION
**Pull request by sum2012**
Colin McRae Rally 2005+ use it.

Do we need to create g_config.ADHOC_CHANNEL ?

### CODE REVIEW

File: Core/HLE/sceUtility.cpp [hunk 1]
```diff
@@ -875,7 +894,7 @@ const HLEFunction sceUtility[] =
 	{0XF3F76017, &WrapI_V<sceUtilityOskGetStatus>,                 "sceUtilityOskGetStatus",                 'i', ""   },
 
 	{0X41E30674, &WrapU_UU<sceUtilitySetSystemParamString>,        "sceUtilitySetSystemParamString",         'x', "xx" },
-	{0X45C18506, nullptr,                                          "sceUtilitySetSystemParamInt",            '?', ""   },
+	{0X45C18506, &WrapU_UU<sceUtilityGetSystemParamInt_>,          "sceUtilitySetSystemParamInt",            'x', "xx"   },
```
Comment (unknownbrackets) [inline Core/HLE/sceUtility.cpp]: Why did you name it Get when it's Set?

-[Unknown]

File: Core/Dialog/PSPDialog.h [hunk 2]
```diff
@@ -26,9 +26,11 @@ class PointerWrap;
...
 #define SCE_UTILITY_DIALOG_RESULT_ABORT        2
 
-const int SCE_ERROR_UTILITY_INVALID_STATUS     = 0x80110001;
-const int SCE_ERROR_UTILITY_INVALID_PARAM_SIZE = 0x80110004;
-const int SCE_ERROR_UTILITY_WRONG_TYPE         = 0x80110005;
+const int SCE_ERROR_UTILITY_INVALID_STATUS      = 0x80110001;
+const int SCE_ERROR_UTILITY_INVALID_PARAM_SIZE  = 0x80110004;
+const int SCE_ERROR_UTILITY_WRONG_TYPE          = 0x80110005;
+const int ERROR_UTILITY_INVALID_ADHOC_CHANNEL   = 0x80110104;
+const int ERROR_UTILITY_INVALID_SYSTEM_PARAM_ID = 0x80110103;
```
Comment (unknownbrackets) [inline Core/Dialog/PSPDialog.h]: Is there a reason to have no SCE_ prefix on these?

-[Unknown]

File: Core/HLE/sceUtility.cpp [hunk 3]
```diff
@@ -690,6 +690,25 @@ static u32 sceUtilityGetSystemParamString(u32 id, u32 destaddr, int destSize)
 	return 0;
 }
 
+static u32 sceUtilityGetSystemParamInt_(u32 id, u32 value)
+{
+	        switch (id) {
```
Comment (unknownbrackets) [inline Core/HLE/sceUtility.cpp]: A lot of these lines use spaces for indentation.

I recommend installing editorconfig:
https://marketplace.visualstudio.com/items?itemName=EditorConfigTeam.EditorConfig

And then simply reindent this section.  Tabs for indent, spaces for alignment.

-[Unknown]

**Review by unknownbrackets — CHANGES_REQUESTED** [review]

File: Core/HLE/sceUtility.cpp (hunk 1 above)
Comment (sum2012) [inline Core/HLE/sceUtility.cpp]: There is a another function same name ,so I add _

File: Core/Dialog/PSPDialog.h (hunk 2 above)
Comment (sum2012) [inline Core/Dialog/PSPDialog.h]: Copy from jpcsp source

File: Core/HLE/sceUtility.cpp (hunk 3 above)
Comment (hrydgard) [inline Core/HLE/sceUtility.cpp]: Tip: Can autoindent the selected code with Ctrl+K, Ctrl+F .

Comment (sum2012) [inline Core/HLE/sceUtility.cpp]: @hrydgard Very Thanks tip

**Comment by sum2012** [comment]
done

**Comment by hrydgard** [comment]
Fixed the get/set confusion and merged manually, so closing.

**Comment by sum2012** [comment]
oop,thanks

**Pull request closed without merging**